packages/api-derive/test/e2e/promise.spec.ts

```diff
@@ -100,4 +107,35 @@ describe.skip('derive e2e', () => {
       done();
     });
   });
+
+  describe('verifies derive.staking.unlocking',() => {
+    const BOND_VALUE = 10;
+    const UNBOND_VALUE = 1;
+    const ALICE_STASH = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY';
+    const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+    const keyring = testKeyring();
+    const aliceStashPair = keyring.getPair(ALICE_STASH);
+    const alicePair = keyring.getPair(ALICE);
+
+    it('bondsExtra funds for Alice Stash', () => {
+      return api.tx.staking.bondExtra(BOND_VALUE)
+        .signAndSend(aliceStashPair, ({ events = [], status }) => {
+          expect(status.type).toMatch(/Flinalized/i);
```

Comment:
Finalized doesn't come in in the first round, so you need some hoops, such as - https://github.com/polkadot-js/api/pull/960/files#diff-a2134f09bc132a8d3d7e01270da62a86R53